composer.json

```diff
@@ -17,7 +17,9 @@
     ],
     "require": {
         "enqueue/enqueue-bundle": "^0.8.0",
-        "symfony/messenger": "^4.1@dev"
+        "symfony/messenger": "^4.1@dev",
```

Comment:
Now that 4.1 is officially released, could be changed for `"symfony/messenger": "^4.1"` (or `"symfony/messenger": "~4.1.0"` maybe, as still experimental)

Reply:
ok